lib/wasix/src/os/console/mod.rs

```diff
@@ -205,7 +205,34 @@ impl Console {
             }
         };
 
-        let wasi_opts = webc::metadata::annotations::Wasi::new(prog);
+        let command = if let Some(entrypoint) = pkg.entrypoint_cmd.as_deref() {
+            pkg.get_command(entrypoint).unwrap()
+        } else {
+            match pkg.commands.as_slice() {
+                [] => {
+                    return Err(SpawnError::Other(
+                        anyhow::anyhow!("The WEBC file doesn't contain any executable commands")
+                            .into(),
+                    ))
+                }
+                [one] => one,
+                [first, ..] => first,
```

Comment:
We should error out here because the situation is ambiguous. 

That's what we do [in the `webc` crate](https://github.com/wasmerio/pirita/blob/f99de1f75309f8f43e3b4585d3a6601bc5225ac3/crates/webc/src/wasmer_package/manifest.rs#L655-L668) when loading a package directly from disk (i.e. `wasmer run .` and tarball conversion).

```rs
/// Infer the package's entrypoint.
fn entrypoint(manifest: &wasmer_toml::Manifest) -> Option<String> {
    if let Some(entrypoint) = &manifest.package.entrypoint {
        return Some(entrypoint.clone());
    }

    if let [only_command] = manifest.commands.as_slice() {
        // For convenience (and to stay compatible with old docs), if there is
        // only one command we'll use that as the entrypoint
        return Some(only_command.get_name().to_string());
    }

    None
}
```

Reply:
We can't error out here because that's what Edge is currently doing and expects.

Reply:
I agree that ideally we would error out here with a misconfiguration error, but for now we need to validate that this won't break anything.